src/attributes/classes.js

```diff
@@ -23,9 +39,7 @@ jQuery.fn.extend( {
 			} );
 		}
 
-		if ( typeof value === "string" && value ) {
-			classes = value.match( rnothtmlwhite ) || [];
```

Comment:
I used this code to make it backward compatible. So if instead of an array a string argument is passed to the method, `value.match( rnothtmlwhite ) || [];`  will assing the string parameters as an array to `classes` which the proceeding code can use it to add or remove classes